### ISSUE TITLE
Warning 'props' is missing in props validation.

### ISSUE THREAD

**Issue by raphaelparent** [yannickcr/eslint-plugin-react#369]
I've looked at the rules and previous issues but I can't see why this is giving me a warning.

I have this pretty basic component.

```
import React from 'react';
import TagHeader from './TagHeader'
import TagItemsContainer from './TagItemsContainer'

export default class TagContainer extends React.Component {

	propTypes: {
		items: React.PropTypes.array.isRequired,
		title: React.PropTypes.string.isRequired
	};

	render() {
		console.log(this);
		return (
			<div className="tag">
				<TagHeader title={this.props.title} />
				<TagItemsContainer items={this.props.items} />
			</div>
		);
	}
}
```

And I get this warning.

```
WARNING in ./src/component/TagContainer.js

/Users/rparent/Documents/CTSProjects/cem/modules/c360-component-tag/src/component/TagContainer.js
  17:34  warning  'title' is missing in props validation  react/prop-types
  18:42  warning  'items' is missing in props validation  react/prop-types

✖ 2 problems (0 errors, 2 warnings)
```

Probably a little something somewhere that I am missing.

I can post more code, let me know what you want to see.

**EDIT -----**

I continued writing my component and now I get 

```
13:23  warning  'items.map' is missing in props validation  react/prop-types
```
When map is the function associated with arrays. For me, this doesn't make any sense.

**Comment by yannickcr** [comment]
Same error as #203, property declaration must be followed by a `=`, not a `:`.

See https://github.com/yannickcr/eslint-plugin-react/issues/203#issuecomment-136501080 for explanation.
